gap/OscarInterface/gap/OscarInterface.gd

```diff
@@ -28,3 +28,9 @@ DeclareProperty( "GroupGeneratorsDefinePresentation", IsGroup );
 # (The methods will be Julia functions.)
 DeclareOperation( "SerializeInOscar", [ IsObject, IsObject ] );
 DeclareConstructor( "DeserializeInOscar", [ IsObject, IsObject, IsObject ] );
+
+############################################################################
+
+# Use GAP operations for the serialization of GAP objects.
+# (The methods will be Julia functions.)
```

Comment:
The comment lines do not fit.
If we want to make a comment then perhaps "for backwards compatibility"?

Reply:
Oops, yes, I just copied the block from above but forgot to update it. Will do that tomorrow